tests/test_data/cir/cir_metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+    "ci_version": 3,
+    "data_version": "0.0.1",
+    "classifier_type": "form_type",
+    "classifier_value": "0001",
+    "guid": "427d40e6-f54a-4512-a8ba-e4dea54ea3dc",
+    "language": "en",
+    "published_at": "2024-07-16T14:26:44.609010Z",
+    "survey_id": "1",
+    "title": "Monthly Wages and Salaries Survey"
+}
```

Comment:
I was expecting a list of metadata objects here?

Reply:
Updated to be a list